sql/plsql/PlSqlParser.g4

```diff
@@ -1993,15 +1993,15 @@ blockchain_row_retention_clause
     ;
 
 blockchain_hash_and_data_format_clause
-    : HASHING USING sha2_512 VERSION v1
+    : HASHING USING sha2_512_quoted VERSION v1_quoted
```

Comment:
Can `sha2_512_quoted` and `v1_quoted` just be inlined? What is the purpose of primitive parser rules?

```suggestion
    : HASHING USING SHA2_512_Q VERSION V1_Q
```